### PR TITLE
chore(IT Wallet): [SIW-962] Add cards onboarding CTA in the new wallet section

### PR DESCRIPTION
**Pull request by mastro993**
## Short description
This PR adds the card onboarding CTA in the new wallet section.

## List of changes proposed in this pull request
- Refactored `WalletHomeScreen` to allow the addition of the CTA
- Added `WalletHomeScreen` component tests

## How to test
From the **Profile** section, enable the **New wallet section** FF.
You should be able to see the CTA in the wallet screen

### CODE REVIEW

**Review by LazyAfternoons — CHANGES_REQUESTED** [review]
Outdated snapshots, can you update them please?

File: ts/features/newWallet/screens/WalletHomeScreen.tsx [hunk 1]
```diff
@@ -19,17 +20,34 @@ const WalletHomeScreen = () => {
     dispatch(idPayWalletGet.request());
   }, [dispatch]);
 
+  if (cards.length === 0) {
+    return (
+      <ScrollView contentContainerStyle={IOStyles.flex}>
+        <WalletPaymentsRedirectBanner />
```
Comment (LazyAfternoons) [inline ts/features/newWallet/screens/WalletHomeScreen.tsx:26]: There's an issue with the banner which doesn't respect the horizontal margins: 

simulator_screenshot_81AC23D0-5014-417D-83DC-B69E9CFB2448

**Review by LazyAfternoons — CHANGES_REQUESTED** [review]

File: ts/features/newWallet/screens/WalletHomeScreen.tsx (hunk 1 above)
Comment (mastro993) [reply ts/features/newWallet/screens/WalletHomeScreen.tsx]: Fixed in https://github.com/pagopa/io-app/pull/5645/commits/9a86bd22eff4f65caad10d037ce6f136201daa39, lmk!

**Review by LazyAfternoons — APPROVED** [review]
LGTM

**Pull request merged**